Packs/Code42/ReleaseNotes/6_0_5.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Code42 Event Collector
+
+- Fixed an issue where not all File Events were fetched.
```

Comment:
- Fixed an issue where not all file events were fetched.

( i would do lower case, as the description in the yml has lower case 'Use the Code42EventCollector integration to fetch file events and audit logs')